src/Histogram.ts

```diff
@@ -75,18 +75,22 @@ export default class Histogram {
   }
 
   // return the bin index of the given data value
-  public static findBin(dataValue: number, dataMin: number, binSize: number, numBins: number): number {
-    let binIndex = Math.floor((dataValue - dataMin) / binSize);
-    // for values that lie exactly on last bin we need to subtract one
-    if (binIndex === numBins) {
-      binIndex--;
-    }
-    return binIndex;
+  public static findBin(dataValue: number, dataMin: number, binSize: number): number {
+    return Math.floor((dataValue - dataMin) / binSize);
```

Comment:
A bit annoying that the same expression as below is also duplicated here, but that was an odd bit of API design that predates this PR. I'd say this ought to be the single place this calculation gets made, but this version also involves a `Math.floor` and so isn't usable in `findFractionalBinOfValue`...